inc/saimacsec.h

```diff
@@ -778,6 +795,15 @@ typedef enum _sai_macsec_sa_attr_t
      */
     SAI_MACSEC_SA_ATTR_MINIMUM_XPN,
 
+    /**
+     * @brief SSCI value for this Secure Association
+     * Used only if SAI_MACSEC_SC_ATTR_MACSEC_XPN64_ENABLE == true.
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_MACSEC_SA_ATTR_MACSEC_SSCI,
```

Comment:
For point-to-point SC, the SC does not survive CA membership change and there is no need for per-SA SSCI.  The only membership change in an established CA is when it goes from 2 members to 1 and the SC should be dismantled.
For point-to-multipoint SC, the only reason SSCI has to change is because SSCI is not part of the key agreement protocol and is derived from SCI in a kludgy way.  Since we designed saimacsec to only support p2p, we decided to avoid the kludge.
If we need to support point-to-multipoint SC, then this change is needed.

Reply:
TX SC is created prior to knowledge of peer SCI even in p2p scenario.  It is not possible to assign SSCI value to SC without knowledge of the peer SCI.  Standard states that SSCI is a parameter of the SA.

Reply:
The sai_create_macsec_sc_fn can be called after the NOS has established SC in both directions.  So it is still a choice.  I have no objection a change to SAI_MACSEC_SA_ATTR_MACSEC_SSCI, but it is would require change to some code that has already been written.